src/packages.builds

```diff
@@ -29,6 +29,12 @@
     <Project Include="..\pkg\Microsoft.NETCore.Targets\Microsoft.NETCore.Targets.builds">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
+    <Project Include="System.Net.Http\pkg\System.Net.Http.builds">
```

Comment:
Why wouldn't this just be automatically includded with:
```xml
     <Project Include="..\pkg\*\*.builds" Condition="'$(SkipManagedPackageBuild)' != 'true'">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
```
Do we want to build this even when SkipManagedPackageBuild is set to true? @weshaggard